ansible/.env.j2

```
@@ -0,0 +1,7 @@
+[Unit]
+Description=My To-Do App
+
+[Service]
+User=ec2-user
+WorkingDirectory=/opt/todoapp
+ExecStart=/home/ec2-user/.local/bin/poetry run flask run --host 0.0.0.0
```

I think you've copied the wrong file into this one!

Copied from your control node:
```suggestion
FLASK_APP=todo_app/app

FLASK_DEBUG=true

SECRET_KEY=secret-key
TRELLO_API_KEY=<key>
TRELLO_API_TOKEN={{ trello_token }}
BOARD_ID=65cb8756241b6ef4a130cda8
TO_DO_LIST_ID=65cb8e035ab1d39df3532134
DONE_LIST_ID=65cb9484fe064787e8c2375a
```
(You are actually correct that the key isn't technically sensitive, but I'm always a little wary of posting it anyway so I redacted that here)